pkgs/development/python-modules/proxy-py/default.nix

```diff
@@ -29,6 +31,21 @@ buildPythonPackage rec {
     hash = "sha256-dA7a9RicBFCSf6IoGX/CdvI8x/xMOFfNtyuvFn9YmHI=";
   };
 
+  patches = [
+    # https://github.com/abhinavsingh/proxy.py/pull/1209
+    (fetchpatch {
+      name = "update-build-dependencies.patch";
+      url = "https://github.com/abhinavsingh/proxy.py/commit/2e535360ce5ed9734f2c00dc6aefe5ebd281cea5.patch";
```

Comment:
This patch looks a bit brittle. Can we confirm that a later released version of `setuptools-scm` won't break this?

Reply:
Nevermind, the next patch removes the weirdness

Reply:
I added a comment to the first patch explaining it is there so that the second one applies cleanly. Thank you for checking and pointing it out.